### PR TITLE
[CODE] Handle display of Classes with multiple super-classes

### PR DESCRIPTION
**Pull request by Dataliberate**
1. Make Thing appear at end of ‘Properties from’ listing
2. Remove Inheritance display (breadcrums) from label display, replace with separate listing below for all inheritance path(s)

### CODE REVIEW

File: api.py [hunk 1]
```diff
@@ -544,28 +552,69 @@ def makeLinksFromArray(self, nodearray, tooltip=''):
...
-                    self.write(" :: ")
-                elif ind > 0:
-                    self.write(" &gt; ")
-                if ind == 1:
-                    self.write("<span property=\"rdfs:label\">")
-                if ind == 0:
-                    self.write("</span>")
+        self.write(node.id)
         self.write("</h1>")
+        self.BreadCrums(node)
```
Comment (danbri) [inline api.py]: "crumbs" is more conventional spelling

Comment (sesuncedu) [inline api.py]: Penfold, shush.

**Comment by danbri** [comment]
I forgot the bit where I say "thanks!" first...

**Comment by Dataliberate** [comment]
No problem - both with the thanks and the correcting to reflect the
convention.   I should have gotten around to doing this much sooner,
however the founding of sdo-gozer seemed a good moment to pick.

On 5 February 2015 at 13:57, Dan Brickley <notifications@github.com> wrote:

> I forgot the bit where I say "thanks!" first...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/schemaorg/schemaorg/pull/320#issuecomment-73104402>.
>



-- 
Richard Wallis
Founder, Data Liberate
http://dataliberate.com
Tel: +44 (0)7767 886 005

Linkedin: http://www.linkedin.com/in/richardwallis
Skype: richard.wallis1
Twitter: @rjw

**Comment by Dataliberate** [comment]
Superseded by pull #567

**Comment by danbri** [comment]
Thanks for the fix Richard. For review see http://sdo-ganymede.appspot.com/LocalBusiness and below

**Comment by Dataliberate** [comment]
http://sdo-ganymede.appspot.com/AutoPartsStore

Looks impressive ?

~Richard


On 8 Jun 2015, at 16:33, Dan Brickley <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for the fix Richard. For review see http://sdo-ganymede.appspot.com/LocalBusiness and below

-

**Comment by danbri** [comment]
It does look good! Though we'll have to be careful that combinations of multiple inheritance don't blow up and create screenfulls of these paths :)

**Pull request closed without merging**